osu.Game/Online/API/Requests/Responses/SoloScoreInfo.cs

```diff
@@ -132,8 +129,7 @@ public ScoreInfo ToScoreInfo(RulesetStore rulesets, BeatmapInfo? beatmap = null)
             Rank = Rank,
             Statistics = Statistics,
             Date = EndedAt ?? DateTimeOffset.Now,
-            Hash = "online", // TODO: temporary?
-            HasReplay = HasReplay,
+            Hash = HasReplay ? "online" : string.Empty, // TODO: temporary?
```

Comment:
Feels weird that the hash is associated by whether the score "has replay", but given it's temporary I think it's fine.

Reply:
It's how it previously worked. I'm not changing it here.

Reply:
Right, realised that's the same as what `ScoreInfo` does.